### PR TITLE
Updated README.md to improve the understanding on how to run both server and client

### PR DESCRIPTION
**Pull request by aldomonteiro**
Hi,

I have just added some comments on how to run server and dev simultaneously. An user opened the issue #5 because it is not clear in the README how to handle the server and the client.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -82,9 +73,16 @@ npm install
...
+Open the terminal and go to the folder server/ and run `npm run dev`. The server is gonna start and listen in the port 3333.
+
+Open a new terminal and go to the folder client/ and run `npm dev`. The client is gonna start and listen in the port 3000.
+
+The client is reachable on `localhost:3000/reduxauth`.
+
 |Script|Description|
 |---|---|
 |`npm run dev`| Run development server |
+|`npm dev`| Run development client |
```
Comment (DimiMikadze) [inline readme.md]: it should be `npm run dev` otherwise it won't be able to run dev server

**Comment by DimiMikadze** [comment]
Hey, thanks for PR. I've added small note, please fix it and i'll merge it, thanks!

**Comment by aldomonteiro** [comment]
Hi, I've just fixed the run command in the client side.

**Comment by DimiMikadze** [comment]
Thanks, merged

**Pull request merged**